style.css

```diff
@@ -655,6 +687,221 @@ footer {
   width: 1px;
 }
 
+/* ///////Popup//////// */
+
+.popup {
+  width: 97%;
+  height: fit-content;
+  position: absolute;
+  top: 0;
+  left: 0;
+  margin: 0 auto;
+  float: left;
+  padding: 3% 1.5%;
+  display: none;
+}
+
+.popup .popup_card {
+  width: 80%;
+  position: absolute;
+  top: 0;
+  left: 5%;
+  right: 5%;
+  height: fit-content;
+  margin: 10px auto;
+  background-color: white;
+  padding: 15px;
+  border-radius: 8px;
+  z-index: 2;
+  transform: translate(50%, 50%) scale(0);
+
+  /* box-sizing: border-box; */
+}
+
+.popup .popup_card > * {
+  flex: 1;
+}
+
+.popup .popup_card .project_title {
+  width: 95%;
+  height: auto;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.popup .popup_card .project_title h1 {
+  margin: 0;
+}
+
+.popup .popup_card .project_title:nth-child(1) {
+  flex: 1;
+  text-align: left;
+  cursor: pointer;
+}
+
+.popup .popup_card .project_title:nth-child(2) {
+  flex: 1;
+  text-align: right;
```

Comment:
- [ ] Kindly make sure you stick to the design as much as possible, the spacing(margin, padding) should suppose to be similar to the Figma design.

### Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/224164540-f12032b3-f3d2-4d59-b22f-00f60888c6bd.png)
### 2
![image](https://user-images.githubusercontent.com/97100091/224164745-27748ed1-eca2-45b1-97f3-2dcb7eb129b8.png)


### Figma template :point_down: 
![image](https://user-images.githubusercontent.com/97100091/224164436-503788d6-31fc-488e-8e87-9a1038b6c2f6.png)

### 2

![image](https://user-images.githubusercontent.com/97100091/224165214-cf7645aa-60a0-4d2c-9797-83dfb28a23ee.png)